internal/webserver/webserver.go

```diff
@@ -306,8 +306,15 @@ func (n *kubeFilter) Start(ctx context.Context) error {
 			tlsConfig := &tls.Config{
 				MinVersion: tls.VersionTLS12,
 				ClientCAs:  n.serverOptions.GetCertificateAuthorityPool(),
-				ClientAuth: tls.VerifyClientCertIfGiven,
+				ClientAuth: tls.NoClientCert,
```

Comment:
`tls.NoClientCert` is the default value, according to the `stdlib` package.

```
	// ClientAuth determines the server's policy for
	// TLS Client Authentication. The default is NoClientCert.
	ClientAuth ClientAuthType
```

Reply:
ok, i can remove this line 👍